prototypes/recipe.lua

```diff
@@ -7,162 +7,162 @@ data:extend({
 	{ -- Basic Warehouse
 		type = "recipe",
 		name = "warehouse-basic",
-		enabled = "false",
+		enabled = false,
 		ingredients =
 		{
-			{ "steel-plate", 200 },
-			{ "stone-brick", 40 },
-			{ "iron-stick", 85 },
+			{ type = "item", name = "steel-plate", amount = 200 },
+			{ type = "item", name = "stone-brick", amount = 40 },
+			{ type = "item", name = "iron-stick", amount = 85 },
 		},
 		energy_required = 30,
-		result = "warehouse-basic",
+		results = {{type="item", name="warehouse-basic", amount = 10}},
```

Comment:
Patch would give 10 per craft? 😕

```suggestion
		results = {{type="item", name="warehouse-basic", amount = 1}},
```

Reply:
Yeah, whoops, quick one to fix